src/main/java/com/thevoxelbox/voxelsniper/Message.java

```diff
@@ -86,19 +88,53 @@ public void height() {
     }
 
     /**
-     * Display performer.
+     * Display performer data.
      *
-     * @param performerName
+     * @param placeMethod
+     * @param replaceMethod
+     * @param usePhysics
      */
-    public void performerName(String performerName) {
-        this.snipeData.sendMessage(TextColors.DARK_PURPLE, "Performer: ", TextColors.DARK_GREEN, performerName);
+    public void performerData(PerformBrush.PerformerType placeMethod,
+                              PerformBrush.PerformerType replaceMethod,
+                              boolean usePhysics) {
+        String physics = usePhysics ? "On" : "Off";
+        this.snipeData.sendMessage(
+                TextColors.DARK_PURPLE,
+                "Performers:",
+                TextColors.GREEN,
+                " place=",
+                TextColors.AQUA,
+                performerTypeToString(placeMethod),
+                TextColors.GREEN,
+                " replace=",
+                TextColors.AQUA,
+                performerTypeToString(replaceMethod),
+                TextColors.GREEN,
+                " physics=",
+                TextColors.AQUA,
+                physics);
+    }
+
+    private String performerTypeToString(PerformBrush.PerformerType type) {
+        switch (type) {
```

Comment:
Code smell: Make this a method on the type to return the string instead of making a switch statement.